addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1182,7 +1187,7 @@ var FieldBinaryImage = AbstractFieldBinary.extend({
                     id: JSON.stringify(this.res_id),
                     field: this.nodeOptions.preview_image || this.name,
                     // unique forces a reload of the image when the record has been updated
-                    unique: (this.recordData.__last_update || '').replace(/[^0-9]/g, ''),
+                    unique: (this.recordData.__last_update && field_utils.format.datetime(this.recordData.__last_update) || '').replace(/[^0-9]/g, ''),
```

Comment:
field utils format datetime accepts a false value, and returns a string, so I think that this line can be simplified to

+                    unique: field_utils.format.datetime(this.recordData.__last_update).replace(/[^0-9]/g, ''),